src/components/createAccount.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react'
+import "../styles/createAccount.css"
+
+export function CreateAccount() {
+  const [firstName, setFirstName] = useState('');
+  const [lastName, setLastName] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [admin, setAdmin] = useState('');
+
+  const handleClick=(e) => {
+    const user = {
+      firstName,
+      lastName,
+      email,
+      password,
+      admin
+    };
+    console.log(user);
+    fetch("http://localhost:8082/user/create", {
```

Comment:
Nog zo'n zelfde vraag;
de ene site gebruikt een useEffect voor post requests, de ander niet. Heb je hier bewust gekozen deze niet te gebruiken?

Reply:
Thanks, PR staat alleen nog in Draft, dus ben nog niet klaar haha